src/scripts.js

```diff
@@ -1,21 +1,29 @@
-// import './css/base.scss';
-// import './css/styles.scss';
-//
-//
-// import Pantry from './pantry';
-// import Recipe from './recipe';
-// import User from './user';
-// import Cookbook from './cookbook';
+import './scss/base.scss';
```

Comment:
We don't necessarily need this. I took it out on a previous branch and just hold some sassy examples. 🤪